### PR TITLE
Implemented pages

### PR DESCRIPTION
**Pull request by TrueEdOs**
Created set of pages with production-ready design.

### CODE REVIEW

**Comment by gmltA** [comment]
Link to deployment?

**Comment by TrueEdOs** [comment]
Sorry, here https://czar-dictionary.web.app/

**Comment by TrueEdOs** [comment]
> Link to deployment?

Sorry, here https://czar-dictionary.web.app/

File: login.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    </header>
+
+    <main>
+        <div id="content">
+            <div id="search-container">
+                <i class="fa fa-search search-icon"></i>
+                <input id="search-input" placeholder="One step remains...">
+            </div>
+            <section id="auth-section">
+                    <h1 class="center-header" id="login-helper">Nickname</h1>
```
Comment (gmltA) [inline login.html]:
```suggestion
                    <label class="center-header" id="login-helper" for="login-enter">Nickname</label >

```

File: assets/css/header.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+}
+
+#main-header {
+	font-family: 'Bungee Shade';
+	font-size: 6em;
+  margin-top: -0.5em;
```
Comment (gmltA) [inline assets/css/header.css]: Use `line-height` instead of negative margins.

**Review by gmltA — CHANGES_REQUESTED** [review]

**Review by gmltA — APPROVED** [review]

**Pull request merged**